create_binary.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+set -x
+
+pushd build_calicoctl
+docker build -t calico-build .
+popd
+
+mkdir -p `pwd`/dist
+chmod 777 `pwd`/dist
+
+docker run -v `pwd`/:/code --name pyinstaller calico-build \
```

Comment:
Circle CI fails if I try to remove a docker container, I think because it has docker-in-docker issues. The goal here is to not have docker containers accumulate on my dev machine or on jenkins. Naming them allows me to delete them without grepping shenanigans.